processes/ingest/chicago_isac.py

```diff
@@ -1,33 +1,40 @@
 import json
 import os
-from requests.exceptions import HTTPError, ConnectionError
 
 from ..core import CoreProcess
-from mappings.core import MappingError
 from mappings.chicagoISAC import ChicagoISACMapping
-from managers import WebpubManifest
+from managers import DBManager, ElasticsearchManager, RedisManager, S3Manager, WebpubManifest
 from logger import createLog
 
 logger = createLog(__name__)
 
 class ChicagoISACProcess(CoreProcess):
 
     def __init__(self, *args):
-        super(ChicagoISACProcess, self).__init__(*args[:4])
 
         self.ingestOffset = int(args[5] or 0)
         self.ingestLimit = (int(args[4]) + self.ingestOffset) if args[4] else 5000
         self.fullImport = self.process == 'complete' 
 
-        # Connect to database
+        self.db_manager = DBManager()
+        self.elastic_search_manager = ElasticsearchManager()
+        self.redis_manager = RedisManager()
```

Comment:
I decided to stick with inheritance for now since it would make sense to switch to composition when we decided to make a full transition with the other processes